plugins/chain/plugin.cpp

```diff
@@ -313,6 +323,21 @@ namespace chain {
                 my->loaded_checkpoints[item.first] = item.second;
             }
         }
+
+        if (options.count("store-account-metadata")) {
+            my->store_account_metadata = options.at("store-account-metadata").as<bool>() ?
+                golos::chain::database::store_for_all : golos::chain::database::store_for_no_one;
```

Comment:
Ternary operator is hard to read.
Please, use simple `if`.

Reply:
@afalaleev, it's one of my favourite operators) With proper indenting even nested ternary operators are readable. Furthermore, it allows combine logic in 1 place (variable assignment, argument in function call) instead of 2, which can prevent copy-paste bugs.
So I prefer to discuss before ban it in our code style.

But I agree, `if` fits good in this code, because of `options.at("store-account-metadata")` duplication. And this change makes code a bit simpler.

Reply:
Sometimes ternary operator allows to make code more readable.
I don't add it to ban list in our code style.